src/content/docs/new-relic-control/agent-control/migration.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "Agent Control migration guide"
+metaDescription: "You can reinstall your current instrumentation to manage it through Agent Control"
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+## Migrate the instrumentation to manage it with Agent Control
+If you have a cluster already instrumented with NewRelic you can decide to remove the existing agents and manage them through the Agent Control.
+All features and all configs exposed by the HelmChart of each Agent can still be applied in a unified way through the Agent Control.
+
+### Match the existing instrumentation with the new Agents managed by the AgentControl
+First of all you need to match your existing instrumentation with the supported agents.
+
+In particular you can deploy and manage through the Agent Control:
+  - Most of the Helm Charts included in the `nri-bundle` chart: 
+    - `newrelic-infrastructure`, `nri-kube-events`, `kube-state-metrics`, and `nri-kube-events` now managed through the New Relic Infrastructure AgentType.
+    - `newrelic-prometheus-configurator` now managed through the New Relic Prometheus AgentType
+    - `newrelic-logging` now managed through the Fluent Bit AgentType
+  - And the chart `nr-k8s-otel-collector` now managed through the New Relic OpenTelemetry Collector AgentType 
+
+If you have installed the agents leveraging Helm you need to retrieve the `values.yaml` used during the installation. 
+If you have it stored no action is needed, otherwise you can retrive it from the cluster.
+You can retrieve the values running the following command:
+```sh
+$ helm get values <release-name> --namespace <namespace-name>
+```
+
+On the other hand, if you have instrumented the cluster via manifest you need to find in the corresponding options in each Agent Chart. 
+
+<Callout title="TIP">
+    You can keep in the cluster any agent or component, in particular the ones not supported yet by the Agent Control. 
+    To assure the best experience make sure to keep the same clusterName and point to the same account.
+</Callout>
+
+
+### Create the `values.yaml` config to install the Agent Control
+Start the [Agent Control Installation](/docs/new-relic-control/agent-control/setup/) via Guided install and select all the agents you are interested in.
+Copy locally the `values.yaml`, and before applying it change each Agent config to match the one of the Agents already running in the cluster.
+
+### Disinstall the agents you want to migrate
```

Comment:
I don't know if Disinstall is correct, but isn't Uninstall or Deinstall more commonly used?

Reply:
I reused the old